src/test/scala/firrtlTests/InlineBooleanExpressionsTests.scala

```diff
@@ -0,0 +1,84 @@
+// See LICENSE for license details.
+
+package firrtlTests
+
+import org.scalatest.FlatSpec
+import org.scalatest.Matchers
+import org.scalatest.junit.JUnitRunner
+import firrtl.ir.Circuit
+import firrtl.Parser
+import firrtl.passes.PassExceptions
+import firrtl.annotations._
+import firrtl.passes.{InlineAnnotation, InlineInstances}
+import firrtl.transforms.{InlineBooleanExpressions, NoCircuitDedupAnnotation}
+import logger.{LogLevel, Logger}
+import logger.LogLevel.Debug
+
+/**
+ * Tests inline instances transformation
+ */
+class InlineBooleanExpressionsTests extends LowTransformSpec {
+  def transform = new InlineBooleanExpressions
+  def inline(mod: String): Annotation = {
+    val parts = mod.split('.')
+    val modName = ModuleName(parts.head, CircuitName("Top")) // If this fails, bad input
+    val name = if (parts.size == 1) modName else ComponentName(parts.tail.mkString("."), modName)
+    InlineAnnotation(name)
+  }
```

Comment:
Likely cargo cult include. There's no need to add `InlineAnnotation`s for these tests.